src/components/Image/tests/Image.test.tsx

```diff
@@ -0,0 +1,45 @@
+import * as React from 'react';
+import Image from '..';
+import { mount } from 'enzyme';
+
+describe('source', () => {
+    it('should verify the source type of the string', () => {
+        const imageWrapper = mount(<Image source="string" title ="" alt=""/>);
```

Comment:
Also check value of source props to "string". This applies to all properties for Image.